### PR TITLE
Adding debezium logo as favicon

### PR DESCRIPTION
**Pull request by indraraj**

### CODE REVIEW

File: _layouts/base.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description }}{% endif %}">
   <link rel="alternate" type="application/rss+xml" title="Debezium Blog" href="/blog.atom">
-  <link rel="shortcut icon" type="image/png" href="">
+  <link rel="shortcut icon" type="image/png" href="{{ "/favicon.ico" | prepend: site.baseurl }}" >
```
Comment (gunnarmorling) [inline _layouts/base.html:10]: Do we have the actual favicon file itself already?

Comment (indraraj) [reply _layouts/base.html:10]: > Do we have the actual favicon file itself already?

I add it in this pr.

Comment (gunnarmorling) [reply _layouts/base.html:10]: Doh, missed that, thanks. I was put off by the "+1 -1" marker here on GitHub :)

**Comment by gunnarmorling** [comment]
Applied, thx

**Pull request merged**